### PR TITLE
[en]: update ghippo v0.29.0 release notes

### PR DESCRIPTION
**Pull request by WANG0608GitHub**
sync with #5085

### CODE REVIEW

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 1]
```diff
@@ -3,6 +3,48 @@
...
+
+### Optimizations
+
+- **Optimized** login and password change functions to support username and password encryption
+- **Optimized** binding resource permissions
+
+### Fixes
+
+- **Fixed** SMTP server settings
+- **Fixed** an issue where OEM configuration could not be installed when set to enable
```
Comment (ModetaNiu) [inline docs/en/docs/ghippo/intro/release-notes.md:23]:
```suggestion
- **Fixed** an issue where installation fails when OEM is configured `enable`
```

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 2]
```diff
@@ -3,6 +3,48 @@
...
 so that you can understand the evolution path and feature changes of each version.
 
+## 2024-07-31
+
+### v0.29.0
+
+### Features
+
+- **Added** LDAP server to support LDAPS
+- **Added** installer versions to the platform
```
Comment (windsonsea) [inline docs/en/docs/ghippo/intro/release-notes.md:13]:
```suggestion
- **Added** installer versions to __Global Management__ -> __Settings__ -> __About__
```
zh version shall be modified too.

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 3]
```diff
@@ -3,6 +3,48 @@
...
+
+### Optimizations
+
+- **Optimized** login and password change functions to support username and password encryption
+- **Optimized** binding resource permissions
```
Comment (windsonsea) [inline docs/en/docs/ghippo/intro/release-notes.md:18]:
```suggestion
- **Optimized** tip text about binding resources on __Global Management__ -> __Workspace and Folder__ -> __Resource Group__
```
zh text shall be updated too.

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 4]
```diff
@@ -3,6 +3,48 @@
...
+
+## 2024-06-30
+
+### v0.28.0
+
+### Features
+
+- **Added** OAuth2 Identity Provider generic plugin
+- **Added** username and userLdapFilter fields to LDAP configuration
+- **Added** installation insight-angent prompt in __Billing Config__ in __Operations Management__
```
Comment (ModetaNiu) [inline docs/en/docs/ghippo/intro/release-notes.md:34]:
```suggestion
- **Added** prompt for insight-agent installation in __Billing Config__ in __Operations Management__
```

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 5]
```diff
@@ -3,6 +3,48 @@
...
+### Optimizations
+
+- **Optimized** login and password change functions to support username and password encryption
+- **Optimized** binding resource permissions
+
+### Fixes
+
+- **Fixed** SMTP server settings
+- **Fixed** an issue where OEM configuration could not be installed when set to enable
+- **Fixed** the inconsistency issue of English column names in exported reports in __Operations Management__
```
Comment (windsonsea) [inline docs/en/docs/ghippo/intro/release-notes.md:24]: 英文很好，中文那句不好

File: docs/en/docs/ghippo/intro/release-notes.md (hunk 4 above)
Comment (windsonsea) [inline docs/en/docs/ghippo/intro/release-notes.md:34]:
```suggestion
- **Added** installation insight-angent prompt in __Global Management__ -> __Operations Management__ -> __Billing Config__
```
zh text shall be updated too.

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 6]
```diff
@@ -3,6 +3,48 @@
...
+
+### Features
+
+- **Added** OAuth2 Identity Provider generic plugin
+- **Added** username and userLdapFilter fields to LDAP configuration
+- **Added** installation insight-angent prompt in __Billing Config__ in __Operations Management__
+
+### Optimizations
+
+- **Optimized** user creation to automatically remove spaces before and after the username before saving
```
Comment (windsonsea) [inline docs/en/docs/ghippo/intro/release-notes.md:38]:
```suggestion
- **Optimized** user creation to automatically remove spaces around the username text (if any) before saving
```

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 7]
```diff
@@ -3,6 +3,48 @@
...
+- **Optimized** the binding mechanism of resource search
```
Comment (windsonsea) [inline docs/en/docs/ghippo/intro/release-notes.md:39]:
```suggestion
- **Optimized** the search mechanism of binding resources
```

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 8]
```diff
@@ -3,6 +3,48 @@
...
+- **Optimized** SDK method `ListWorkspaceUsersByPermission()`
```
Comment (windsonsea) [inline docs/en/docs/ghippo/intro/release-notes.md:40]:
```suggestion
- **Optimized** a SDK method `ListWorkspaceUsersByPermission()`
```

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 9]
```diff
@@ -3,6 +3,48 @@
...
+
+### Fixes
+
+- **Fixed** the issue that custom role authorized users can see all workspace
```
Comment (windsonsea) [inline docs/en/docs/ghippo/intro/release-notes.md:44]:
```suggestion
- **Fixed** an issue that authorized users with a custom role can see all workspaces
```
可以全文一致的 style：`**Fixed** an issue...`

File: docs/en/docs/ghippo/intro/release-notes.md [hunk 10]
```diff
@@ -3,6 +3,48 @@
...
+- **Fixed** the issue that the RBAC Access Denied page will appear on the Insight page after closing the browser and reopening it
+- **Fixed** the issue that the report export field name is inconsistent with the page in __Operations Management__
```
Comment (windsonsea) [inline docs/en/docs/ghippo/intro/release-notes.md:46]:
```suggestion
- **Fixed** an issue that the exported report has inconsistent fields with those shown on the page in __Operations Management__
```

**Review by windsonsea — COMMENTED** [review]
部分中文还需要优化一下

**Review by windsonsea — APPROVED** [review]

**Pull request merged**